arm-powerbiembedded/2016-01-29/swagger/powerbiembedded.json

```diff
@@ -0,0 +1,1041 @@
+                        "required": true,
+                        "type": "string"
+                    },
+                    {
+                        "name": "body",
+                        "in": "body",
+                        "description": "Update workspace collection request",
+                        "required": true,
+                        "schema": {
+                            "$ref": "#/definitions/UpdateWorkspaceCollectionRequest"
+                        }
+                    }
+                ],
+                "responses": {
+                    "200": {
+                        "description": "Workspace collection updated successfully",
+                        "schema": {
+                            "$ref": "#/definitions/WorkspaceCollection"
+                        }
+                    },
+                    "default": {
+                        "description": "Default Response. It will be deserialized as per the Error defintion specified in the schema. Exception will be thrown.",
+                        "schema": {
+                            "$ref": "#/definitions/Error"
+                        }
+                    }
+                }
+            },
+            "delete": {
+                "tags": [
+                    "WorkspaceCollections"
+                ],
+                "operationId": "WorkspaceCollection_delete",
+                "description": "The **Delete Power BI Workspace Collection** operation deletes a Power BI Workspace Collection.",
+                "consumes": [
+                    "application/json"
+                ],
+                "produces": [
+                    "application/json"
+                ],
+                "parameters": [
+                    {
+                        "name": "subscriptionId",
+                        "in": "path",
+                        "description": "Azure subscription id",
+                        "required": true,
+                        "type": "string"
+                    },
+                    {
+                        "name": "resourceGroupName",
+                        "in": "path",
+                        "description": "Azure resource group",
+                        "required": true,
+                        "type": "string"
+                    },
+                    {
+                        "name": "workspaceCollectionName",
+                        "in": "path",
+                        "description": "Power BI Embedded workspace collection name",
+                        "required": true,
+                        "type": "string"
+                    },
+                    {
+                        "name": "api-version",
+                        "in": "query",
+                        "description": "Azure API version",
+                        "required": true,
+                        "type": "string"
+                    }
+                ],
+                "responses": {
+                    "202": {
```

Comment:
Since you have a 202, is this a long running operation. If so then please add the `"x-ms-long-running-operation":true` extension over here. Please take alook at all our extensions over here https://github.com/Azure/autorest/blob/master/Documentation/swagger-extensions.md